my.css

```diff
@@ -195,7 +199,10 @@ body {
     background-color: black;
     z-index: -1;
   }
-
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5642160)